File: app/src/main/java/com/article/kotlinsynthetics/MainActivity.java
package com.article.kotlinsynthetics;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);

        getSupportActionBar().setTitle("Kotlin Talk 2");

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, MainFragment.Companion.newInstance())
                    .commitNow();
        }
    }
}

File: app/src/main/java/com/article/kotlinsynthetics/d_kotlin_synthetics/MainActivity.kt
package com.article.kotlinsynthetics.d_kotlin_synthetics

import androidx.appcompat.app.AppCompatActivity

import android.os.Bundle

import com.article.kotlinsynthetics.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)

        supportActionBar!!.title = "Kotlin - sythetics"

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction().replace(R.id.container, MainFragment.newInstance()).commitNow()
        }
    }
}
